.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "printWidth": 80,
+  "semi": true,
+  "singleQuote": false,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "useTabs": false
```

Comment:
Missing the: `arrowParens`, `proseWrap`, `requirePragma`

Reply:
@lipis Those are not present in the JSON schema: http://json.schemastore.org/prettierrc